microsoft-edge/webview2/release-notes.md

```diff
@@ -138,6 +138,387 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
+    * [ICoreWebView2_20::get_FrameId](/microsoft-edge/webview2/reference/win32/icorewebview2_20?view=view=webview2-1.0.2261-prerelease&preserve-view=true#get_frameid)<!--no put-->
+
+* [ICoreWebView2Frame5](/microsoft-edge/webview2/reference/win32/icorewebview2frame5?view=webview2-1.0.2261-prerelease&preserve-view=true)
+    * [ICoreWebView2Frame5::get_FrameId](/microsoft-edge/webview2/reference/win32/icorewebview2frame5?view=webview2-1.0.2261-prerelease&preserve-view=true#get_frameid)<!--no put-->
+
+* [ICoreWebView2FrameInfo2](/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-1.0.2261-prerelease&preserve-view=true)
+    * [ICoreWebView2FrameInfo2::get_FrameId](/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-1.0.2261-prerelease&preserve-view=true#get_frameid)
+    * [ICoreWebView2FrameInfo2::get_FrameKind](/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-1.0.2261-prerelease&preserve-view=true#get_framekind)
+    * [ICoreWebView2FrameInfo2::get_ParentFrameInfo](/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-1.0.2261-prerelease&preserve-view=true#get_parentframeinfo)
+    * [ICoreWebView2FrameInfo2::Invoke](/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-1.0.2261-prerelease&preserve-view=true#invoke)<!-- todo: no "Invoke" member near https://review.learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-win32-preview&branch=pr-en-us-76 - for WinRT, no Invoke member, either: https://review.learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frameinfo?view=webview2-winrt-preview&branch=pr-en-us-46 -->
+
+* [COREWEBVIEW2_FRAME_KIND Enum](/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.2261-prerelease&preserve-view=true#corewebview2_frame_kind)
+
+---
+
+
+<!-- ---------- -->
+* `ExecuteScriptWithResult` provides exception information if the script failed.  `TryGetResultAsString` gets the script execution result as a string rather than as JSON, to make it more convenient to interact with string results:
+
+<!-- todo: make sure lists are sync'd across tabs
+WinRT rendered PR:
+https://review.learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?branch=pr-en-us-46&view=webview2-winrt-preview
+Win32 rendered PR:
+https://review.learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2frameinfo2?view=webview2-win32-preview&branch=pr-en-us-76
+-->
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- todo: test links -->
+
+* `CoreWebView2` Class:
+    * [CoreWebView2.CreateFromComICoreWebView2 Method](/dotnet/api/microsoft.web.webview2.core.corewebview2.createfromcomicorewebview2?view=webview2-dotnet-1.0.2261-prerelease&preserve-view=true)

```

Comment:
Delete CreateFromComICoreWebView2?  It's not listed in WinRT or Win32 tab, not found in WinRT near 
https://review.learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-preview&branch=pr-en-us-46
@peiche-jessica @victorhuangwq

Reply:
Comment is Outdated; surrounding Markdown not the latest.
Latest commits/ fresh Files Changed tab: https://github.com/MicrosoftDocs/edge-developer/pull/2968/files

Reply:
latest has a hidden todo comment

Reply:
closed the dup comment https://github.com/MicrosoftDocs/edge-developer/pull/2968/files/36830a1396281068c49d41752f487c094658d651#r1423280779

Reply:
I agree that we should delete it. It's not related to execute script, and is not found in winrt either

Reply:
deleted